apps/search-poc/src/index.tsx

```diff
@@ -7,7 +7,7 @@ import { NexusProvider } from '@bbp/react-nexus';
 import { setUpSession, setToken } from './utils/auth';
 import Header from './components/Header';
 import SparqlView from './views/Sparql';
-import DetailsView from './views/Details';
+import DetailsContainerView from './views/DetailsContainer';
```

Comment:
Not sure I get why this file has been renamed to `DetailsContainer`... We want to create a new view called `Details View` so why not simply have a `Details` component in the `View` folder?